drivers/bmp180/bmp180.c

```diff
@@ -234,7 +234,7 @@ static int _read_up(const bmp180_t *dev, int32_t *output)
         return -1;
     }
 
-    *output = ((up[0] << 16) | (up[1] << 8) | up[2]) >> (8 - OVERSAMPLING);
```

Comment:
I'm not sure if the compiler will do two casts here (first to 16 and then to 32 to do the OR)... I suggest to set the second casting to `uint32_t` as well.

Reply:
yes, was thinking the same, too - but it didn't fail with `uint16_t` according to Murdock 😉 I'll change it anyway to `uint32_t`, i.e. for consistency with bh1750fvi.